library/src/main/java/org/xmtp/android/library/SigningKey.kt

```diff
@@ -19,9 +19,31 @@ import java.util.Date
 interface SigningKey {
     val address: String
 
-    suspend fun sign(data: ByteArray): SignatureOuterClass.Signature?
+    // If this signing key is a smart contract wallet
+    val isSmartContractWallet: Boolean
+        get() = false
 
-    suspend fun sign(message: String): SignatureOuterClass.Signature?
+    // Default chainId value set to 1
```

Comment:
I'm not sure about this, but maybe we don't want a default chain ID.

If I'm signing in with an EOA, we have no idea what chain I'm really using. If I'm on Base, it's not really fair to assume that I'm using Ethereum.

It could be safer to have the field default to null, and if you have a null chain ID on a SCW to error out at runtime.